pootle/apps/pootle_app/models/permissions.py

```diff
@@ -160,10 +162,10 @@ class Meta:
     user = models.ForeignKey(settings.AUTH_USER_MODEL, db_index=True)
     directory = models.ForeignKey('pootle_app.Directory', db_index=True,
                                   related_name='permission_sets')
-    positive_permissions = models.ManyToManyField(Permission, db_index=True,
-            related_name='permission_sets_positive')
-    negative_permissions = models.ManyToManyField(Permission, db_index=True,
-            related_name='permission_sets_negative')
+    positive_permissions = models.ManyToManyField(
+        Permission, db_index=True, related_name='permission_sets_positive')
+    negative_permissions = models.ManyToManyField(
+        Permission, db_index=True, related_name='permission_sets_negative')
```

Comment:
```python
    positive_permissions = models.ManyToManyField(
        Permission,
        db_index=True,
        related_name='permission_sets_positive',
    )
    negative_permissions = models.ManyToManyField(
        Permission,
        db_index=True,
        related_name='permission_sets_negative',
    )
```

See http://docs.translatehouse.org/projects/pootle/en/latest/developers/styleguide.html#fields-in-models-and-forms